src/quo2/components/loaders/skeleton_list/view.cljs

```diff
@@ -0,0 +1,106 @@
+(ns quo2.components.loaders.skeleton-list.view
+  (:require [quo2.theme :as theme]
+            [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [react-native.masked-view :as masked-view]
+            [react-native.reanimated :as reanimated]
+            [quo2.components.loaders.skeleton-list.style :as style]
+            [reagent.core :as reagent]
+            [quo2.components.loaders.skeleton-list.constants :as constants]))
+
+(defn skeleton-label
```

Comment:
why we are adding accessibility id on each skeleton item? we probably only need add to whole skeleton list.

Reply:
yep, if it's just for testing, we can probably avoid testing to that degree and just do a check of the outer container

Reply:
Thanks for this suggestion, guys! Is way cleaner and faster now. I also refactor to have more clean test descriptions